root.spec

```diff
@@ -3,8 +3,8 @@
 ## INITENV +PATH PYTHON3PATH %{i}/lib
 ## INITENV SET ROOTSYS %{i}
 
-%define tag d9ead8a7d0a08238c33e4920ff0584097904981b
-%define branch cms/v6-22-00-patches/0c7eac6
+%define tag ae63a2f7f3ad0ec5b3dcd932178b7144ced7786c
+%define branch cms/v6-22-00-patches/c25f7e5
```

Comment:
@mrodozov  what is this branch cms/v6-22-00-patches/c25f7e5 ? c25f7e5 should be a commit from https://github.com/root-project/root/commits/v6-22-00-patches but I do not see it there

Reply:
@mrodozov , please use a valid commit from root branch otherwise automatic update tool will break

Reply:
it's fixed